nntrainer/include/tensor.h

```diff
@@ -302,6 +302,13 @@ class Tensor {
    */
   Tensor divide(Tensor const &m) const;
 
+  /**
+   * @brief    Tensor power Element by Element
+   * @param[in] float Divisor Tensor
+   * @retval Calculated Tensor
+   */
+  Tensor pow(float m) const;
```

Comment:
does it give us better accuracy?

Reply:
The validation result says it is equivalent to the very small tolerance. I made this for checking purposes but decided leave here as it is a basic operation.